app/views/ati_network/showcase/_impact_board.html.erb

```diff
@@ -0,0 +1,496 @@
+          <div class="modal-header">
+            <h3 class="modal-title" id="modal-title-2">How access to information can help us understand AI decision making</h3>
+            <button class="modal-close modal-close--just-icon" aria-label="Close">×</button>
+          </div>
+          <div class="modal-body">
+            <div class="video-container">
+              <iframe tabindex="0" width="560" height="315" src="https://www.youtube.com/embed/ilgE2RMx9jQ?enablejsapi=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="gallery-grid--card gallery-grid--card">
+      <div class="card--header-badge"><span class="badge--flag">🌐</span>International</div>
+      <div class="modal-button card-thumbnail" tabindex="0" role="button" aria-label="Open video: How access to information can help us understand AI decision making">
+        <div class="video-thumbnail">
+          <div class="youtube-svg"></div>
+          <img src="https://img.youtube.com/vi/ilgE2RMx9jQ/maxresdefault.jpg" loading="lazy" role="presentation" aria-hidden="true">
+        </div>
+      </div>
+      <div class="modal-content modal-hidden" aria-hidden="true">
+        <div class="modal-container" role="dialog" aria-modal="true" aria-labelledby="modal-title-2">
+          <div class="modal-header">
+            <h3 class="modal-title" id="modal-title-2">How access to information can help us understand AI decision making</h3>
+            <button class="modal-close modal-close--just-icon" aria-label="Close">×</button>
+          </div>
+          <div class="modal-body">
+            <div class="video-container">
+              <iframe tabindex="0" width="560" height="315" src="https://www.youtube.com/embed/ilgE2RMx9jQ?enablejsapi=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <a href="https://automatingsociety.algorithmwatch.org/wp-content/uploads/2020/12/Automating-Society-Report-2020.pdf" class="gallery-grid--card gallery-grid--card--report-cover" aria-label="Visit Automating Society Report 2020">
+      <div class="card--header-badge"><span class="badge--flag">🇪🇺</span>Cross Border</div>
+      <%= image_tag('ati_network/showcase/automating-society-report.jpg', alt: _('Automating Society Report 2020'), :class => "card--image", loading: 'lazy') %>
+    </a>
+
+    <div class="gallery-grid--card">
+      <div class="card--header-badge"><span class="badge--flag">🇫🇷</span>France</div>
```

Comment:
Somehow this is showing france here (in the diff) but europe on the actual site

<img width="366" height="326" alt="Image" src="https://github.com/user-attachments/assets/d866052c-18c3-424e-82d9-a7e08d75ed93" />